src/Misc/decorators.ts

```diff
@@ -247,6 +247,16 @@ export class SerializationHelper {
                 destination.animations.push(animation.serialize());
             }
         }
+
+        if (source.animationGroups)

```

Comment:
animation groups are only hosted by the scene. They cannot be on an object

Reply:
I understand. I think i should just modify the sceneSerializer instead :)